### ISSUE TITLE
Mobi Error when creating new containers

### ISSUE THREAD

**Issue by snpendse** [Open-Systems-Pharmacology/MoBi#517]
I am attempting to create a new spatial structure for a custom PBPK model. Everytime I create a new container, I get the following error. 

Application:
MoBi® 9.1.1

Object reference not set to an instance of an object.

Stack trace:
```
at OSPSuite.UI.Controls.UxTreeView.setNodeIcon(TreeListNode node, ApplicationIcon icon)
   at OSPSuite.UI.Controls.UxTreeView.addNodeToNodeCollection(TreeListNode parentNode, ITreeNode node)
   at OSPSuite.UI.Controls.UxTreeView.<>c__DisplayClass42_0.<AddNode>b__0()
   at OSPSuite.UI.Controls.UxTreeList.DoWithinBatchUpdate(Action action)
   at OSPSuite.UI.Controls.UxTreeView.AddNode(ITreeNode nodeToAdd)
   at OSPSuite.Utility.Extensions.EnumerableExtensions.Each[T](IEnumerable`1 list, Action`1 action)
   at MoBi.UI.Views.SelectReferenceView.<>c__DisplayClass14_0.<addNodes>b__0()
   at OSPSuite.UI.Controls.UxTreeList.DoWithinBatchUpdate(Action action)
   at MoBi.UI.Views.SelectReferenceView.addNodes(IEnumerable`1 nodes, Boolean clear)
   at MoBi.UI.Views.SelectReferenceView.AddNodes(IEnumerable`1 nodes)
   at MoBi.UI.Views.SelectReferenceView.AddNode(ITreeNode node)
   at MoBi.Presentation.Presenter.SelectReferencePresenterBase.<>c__DisplayClass54_0.<Handle>b__0(ITreeNode parentNode)
   at OSPSuite.Utility.Extensions.EnumerableExtensions.Each[T](IEnumerable`1 list, Action`1 action)
   at MoBi.Presentation.Presenter.SelectReferencePresenterBase.Handle(AddedEvent eventToHandle)
   at OSPSuite.Utility.Events.EventPublisher.<>c__DisplayClass4_1`1.<PublishEvent>b__1()
   at OSPSuite.Utility.Exceptions.ExceptionManagerBase.Execute(Action action)
```

**Comment by msevestre** [comment]
Thanks for reporting @snpendse 
I'll see if I can reproduce and get back to you

**Comment by msevestre** [comment]
@snpendse I cannot reproduce using the latest version of MoBi
  
![image](https://user-images.githubusercontent.com/1041237/96494793-d3621b00-1214-11eb-83d5-b9e0b0441141.png)

can you give us please a step by step instructions on how to reproduce?

**Comment by snpendse** [comment]
I am not able to reproduce the error now. Earlier, I was tried containers from the human 2 pore template spatial structure which gave me a similar error. Then it continued to give me errors every time a created a new container, organ or neighborhood. However I am not able to reproduce the problem now. I setup a new issue if i run into it again. Thanks for checking into it right away

**Comment by msevestre** [comment]
Awesome. Thanks for your feedback. Closing for now. Please reopen if you manage to reproduce

**Comment by moPellowe** [comment]
I am currently using version 9.1, but I got a similar error that I was able to reproduce. I'm not sure whether this would happen in version 10 as well. Here's what I did:
- Create a new MoBi session (Amount-based)
- Create a new Spatial Structure with Container type: Other
- Go to Diagram (which is strangely empty)
- Right-click and Create Top Container, which should produce the error

Note that this error can be avoided by either creating a spatial structure that is another container type or creating the container using the Tree view.
